frontend/src/theme/canada.js

```diff
@@ -100,6 +100,13 @@ export default extendTheme({
       800: '#153E75',
       900: '#0D467D',
     },
+    red: {
+      500: '#e50003',
+    },
+    teal: {
+      500: '#007984',
+      600: '#007072',
+    },
```

Comment:
We should be able to keep these link colors as is and use darker colors where needed (default teal.600 for white background, teal.700 for guidance tags)